### PR TITLE
chore: attempt to fix Example app on iOS

### PR DESCRIPTION
**Pull request by Simek**
This PR attempt to fix iOS Example app build. So far I was able to fix XCode errors and build, however the Metro fails to build/detect the app right now. Further work and debug is required.

Changes included in this PR:
* Swift bumped from `5.0` to `5.2` and target updated to iOS `11.0` (as described in Vision Camera Troubleshooting - https://mrousavy.github.io/react-native-vision-camera/docs/guides/troubleshooting/#ios)
* removed `tvOS` targets from XCode Workspace
* `File.swift` moved to the XCode Workspace
* added missing `babel-plugin-module-resolver` dependency
* updated used Vision Camera to fix on of XCode errors (`usePixelAspecRatio`)
* regenerated Pods

Currently the example iOS app build correctly, but Metro fails to detect/build the React app inside.

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ export default function App() {
   }, []);
 
   const frameProcessor = useFrameProcessor((frame) => {
+    'worklet';
     const labels = labelImage(frame);
     _log(`Labels: ${labels.join(', ')}`);
```
Comment (mrousavy) [inline example/src/App.tsx:26]:
```suggestion
    console.log(`Labels: ${labels.join(', ')}`);
```

**Comment by Simek** [comment]
Finally I was able to run the example app on the device. Besides the FrameProcessor error, which might be device specific, the example seems to work quite fine.

### Preview

image

**Comment by Simek** [comment]
The error is now gone, here is the sample output from log:
```
2021-08-20 14:15:03.604379+0200 VisionCameraImageLabelerExample[11005:2264801] [javascript] 'Labels: ', [ { confidence: 0.7398267388343811, label: 'Pattern' },
  { confidence: 0.5818111896514893, label: 'Food' } ]
2021-08-20 14:15:03.967015+0200 VisionCameraImageLabelerExample[11005:2264801] [javascript] 'Labels: ', [ { confidence: 0.7398267388343811, label: 'Pattern' },
  { confidence: 0.5561060905456543, label: 'Food' },
  { confidence: 0.5026870965957642, label: 'Metal' } ]
2021-08-20 14:15:04.342708+0200 VisionCameraImageLabelerExample[11005:2264801] [javascript] 'Labels: ', [ { confidence: 0.696117103099823, label: 'Pattern' },
  { confidence: 0.6224076151847839, label: 'Metal' } ]
2021-08-20 14:15:04.669567+0200 VisionCameraImageLabelerExample[11005:2264801] [javascript] 'Labels: ', [ { confidence: 0.7111600637435913, label: 'Pattern' },
  { confidence: 0.5485353469848633, label: 'Metal' } ]
```

**Comment by mrousavy** [comment]
Amazing job @Simek! 🖤🖤🖤

**Comment by rodgomesc** [comment]
it works pretty well if you just put a `console.log` but when try to set a state using `runOnJS(setThing)(scannedThing);`, app crashes immediately (only on IOS), it's seems to be a old bug related here https://github.com/mrousavy/react-native-multithreading/issues/21#issue-895003064

image

**Pull request merged**